drivers/clock_control/clock_control_lpc11u6x.c

```diff
@@ -0,0 +1,400 @@
+/*
+ * Copyright (c) 2020, Seagate Technology LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT nxp_lpc11u6x_syscon
+
+#include <devicetree.h>
+#include <device.h>
+
+#include <drivers/clock_control/lpc11u6x_clock_control.h>
+#include <drivers/pinmux.h>
+
+#include "clock_control_lpc11u6x.h"
+
+#define DEV_CFG(dev)  ((const struct lpc11u6x_syscon_config *) \
+		      ((dev)->config_info))
+
+#define DEV_DATA(dev) ((struct lpc11u6x_syscon_data *) \
+		      ((dev)->driver_data))
+
+static void syscon_power_up(struct lpc11u6x_syscon_regs *syscon,
+			    u32_t bit, bool enable)
+{
+	if (enable) {
+		syscon->pd_run_cfg = (syscon->pd_run_cfg & ~bit)
+			| LPC11U6X_PDRUNCFG_MASK;
+	} else {
+		syscon->pd_run_cfg = syscon->pd_run_cfg | bit
+			| LPC11U6X_PDRUNCFG_MASK;
+	}
+}
+
+static void syscon_set_pll_src(struct lpc11u6x_syscon_regs *syscon,
+			       u32_t src)
```

Comment:
The zephyr-specific integer types were recently deprecated in #25679. Please rebase and update to use C99 integer types